### ISSUE TITLE
Theme problems

### ISSUE THREAD

**Issue by frankzen** [jun7/rox-filer#202]
I installed a new distro on my machine today - Fedora 31...so I dl'ed Rox and compiled it again as the version in Fedora is OLD.
I ma having a problem with theming the background of the filer. When con view is on, the background is black as I want is to be. But when the list view is active, only part of the filer background is black.

Any ideas as to what's going on. ??

![2019-12-27_142410_283244459](https://user-images.githubusercontent.com/717662/71529952-6d875a00-28b5-11ea-93ba-e811fafb4d34.png)
![2019-12-27_142353_504135287](https://user-images.githubusercontent.com/717662/71529957-76782b80-28b5-11ea-915b-74fc67c7bff9.png

I have  .gtkrc-2.0 file in /home/frank

**Comment by jun7** [comment]
I think the theme you using having color for columns.
Did you try other themes? Which theme are you using?

**Comment by frankzen** [comment]
I have tried various themes..they all seem to result in the same thing. I have a .gtkrc-2.0 file in my home directory, but Rox seems to ignore it while it's in list mode. But this only happens when I am in Fedora. In Debian the background of the filer remains black in any mode. Something weird about Fedora even though it's the same version of the filer in both cases. I didn't even know you could theme columns in different colors!!

**Comment by jun7** [comment]
I installed bluebird that gtk2 theme of fedora 31 and not reproduced on arch linux.
So themes do not effect this problem.
Hmm, seems gtk2's versions are same. I have no clue.

**Comment by frankzen** [comment]
The strange thing is the problem does not exist under XFCE. but does exist under any other window manager, like IceWm or Fluxbox.  I don't know what the difference is? I even tried loading xfconfd in my icewm startup file but it made no difference. Every time I left click on the  list on the toolbar, I get that weird screen with black and white columns. If I right click on list it cycles through all the different display types, with black backgrounds even under IceWm or Fluxbox.

**Comment by step-** [comment]
@frankzen Is Options > Colours > Colour files based on their type ON ? I can't tell from the picture. If ON then try to disable it.

**Comment by frankzen** [comment]
@step Yes it was on, but turning it off doesn't make any difference. I have learned to live with it...I don't use list mode often.

**Comment by step-** [comment]
All I know is that if that option is enabled then rox will not honor the gtk list widget background color as set by the gtk theme.  I don't know anything about "column" color.
